.github/gh_runner_dependencies.sh

```diff
@@ -46,7 +46,8 @@ tfenv install 1.0.0
 tfenv install 1.0.1
 tfenv install 1.0.6
 tfenv install 1.1.3
-tfenv use 1.1.3
+tfenv install 1.2.0
+tfenv use 1.2.0
```

Comment:
do we also manually add these?

Reply:
So I was lazy to SSH into the VMs and set it up. So I added the install commands as part of the CI yamls and removed them after letting it run once: https://github.com/GoogleCloudPlatform/anthos-samples/pull/319/commits/c991267dfe7be7c3ed15db597038360f7e779af2

I am not sure which yaml files you are referring to here!!!

Reply:
(sorry i should've specified!)
these yaml files: 
- .github/workflows/ci_main_branch.yaml
- .github/workflows/ci_main_branch.yaml

i see that the tfenv uses 1.1.3 in the above files in the PR